config/intellij-idea-inspections.xml

```diff
@@ -105,9 +105,14 @@
     <inspection_tool class="AssignmentUsedAsCondition" enabled="true" level="ERROR" enabled_by_default="true" />
     <!-- we are ok to use auto-boxing as we use modern java -->
     <inspection_tool class="AutoBoxing" enabled="false" level="ERROR" enabled_by_default="false" />
-    <!-- we can not use try-with-resource till we switch to jacoco for code coverage -->
-    <inspection_tool class="AutoCloseableResource" enabled="false" level="ERROR" enabled_by_default="false" />
-    <!-- we are ok to use auto-unboxing as we use modern java -->
+    <inspection_tool class="AutoCloseableResource" enabled="true" level="ERROR" enabled_by_default="true">
+        <!-- These classes do not contain any resources that need to be closed. -->
+        <option name="ignoredTypes" value="java.io.ByteArrayOutputStream,java.util.stream.Stream,java.util.stream.IntStream,java.util.stream.LongStream,java.util.stream.DoubleStream,com.puppycrawl.tools.checkstyle.internal.utils.CloseAndFlushTestByteArrayOutputStream" />
+        <!-- False positives for the Closable returned from a builder method (which return "this").
+             Mock stubs shouldn't be closed. -->
+        <option name="METHOD_MATCHER_CONFIG" value="java.util.Formatter,format,java.io.Writer,append,com.google.common.base.Preconditions,checkNotNull,org.hibernate.Session,close,java.io.PrintWriter,printf,org.mockito.Mockito,mock|verify,org.mockito.stubbing.Stubber,when" />
```

Comment:
same.

Reply:
if this is not obvious how to make, we can postpone it for later.